topdown/eval.go

```diff
@@ -139,10 +142,17 @@ func (e *eval) traceEnabled() bool {
 	return e.tracer != nil && e.tracer.Enabled()
 }
 func (e *eval) eval(iter evalIterator) error {
-	return e.evalExpr(0, iter)
+	return e.evalExpr(iter)
 }
 
-func (e *eval) evalExpr(index int, iter evalIterator) error {
+func (e *eval) evalNext(iter evalIterator) error {
```

Comment:
I think we should rename this to `next` or `step` and put it beside the `child` and `closure` helpers above in this file.

Reply:
Done